content/en/docs/setup/install/standalone-operator/index.md

```diff
@@ -30,14 +30,14 @@ instead, which is a stable feature.
     $ kubectl apply -f https://preliminary.istio.io/operator.yaml
     {{< /text >}}
 
-1. (Optionally) change profiles from the default profile to one of the following profiles:
+1. (Optionally) change profiles from the demo profile to one of the following profiles:
```

Comment:
What profile is installed on line 30?

This second step doesn't really make sense, does it? First operator.yaml is used to install istio, and then here we optionally reinstall?

Reply:
it is the demo profile when this PR gets in.

Reply:
i kept this step in case folks want to switch to a non demo profile then back to demo profile.

Reply:
Shouldn't line 30 then be operator-profile-demo.yaml? If not, what's the difference between them?

Reply:
@frankbu the demo profile (which is a CR) is bundled in the operator.yaml.  To switch profiles, simply changing the profile gets things moved around.